### DISCUSSION TITLE
Github Repo: Public or Private?

### DISCUSSION THREAD

**Discussion by Underdoge** [msusdev/Innovation-Challenge#4] [Q&A] [answered]
I'm assuming our Github repository can be created with "Private" visibility specially during development, but we should change the visibility to "Public" before/when we submit our final project, correct?

**Comment by MatthewCalder-msft** [comment, marked as answer]
Correct, it will need to be public so the judges can evaluate the project. But totally fine to keep it "private" while you get things together.

**Comment by Underdoge** [comment]
@MatthewCalder-msft thanks!
